lib/js/jsre.nim

```diff
@@ -44,3 +45,7 @@ func test*(self: RegExp; pattern: cstring): bool {.importjs: "#.test(#)".}
 
 func toString*(self: RegExp): cstring {.importjs: "#.toString()".}
   ## Returns a string representing the RegExp object.
+
+func `$`*(self: RegExp): string = $toString(self)
+
+func len*(self: RegExp): int = self.source.len
```

Comment:
i don't think we need this, since its code can be inlined; the RegExp object contains many fields, it's unclear why special case the length of the source field; but it's ok to add a line in runnableExamples mentioning:
`assert jsregex.source.len == 5`